examples/lines_bars_and_markers/eventcollection_demo.py

```diff
@@ -32,16 +32,16 @@
 # plot the data
 fig = plt.figure()
 ax = fig.add_subplot(1, 1, 1)
-ax.plot(xdata1, ydata1, 'r', xdata2, ydata2, 'b')
+ax.plot(xdata1, ydata1, xdata2, ydata2)
```

Comment:
I would make the colors explicit here. We want the lines and the EventCollection to have the same color, and that shouldn‘t implicitly rely on the used color cycle.